pkgs/development/compilers/boo/default.nix

```diff
@@ -41,5 +41,7 @@ in stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "The Boo Programming Language";
     platforms = platforms.linux;
+    hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
I don't think setting hydraPlatforms does anything useful here. (Your "meta = broken" is all that's needed.)